figo/Config.php

```diff
@@ -42,7 +42,7 @@ class Config {
     /**
      * @var string Version of this SDK, used in user agent for API requests
      */
-    public static $SDK_VERSION = '1.3.0';
+    public static $SDK_VERSION = '2.0.0';
```

Comment:
Version should be 3.0.0!

Reply:
You are right! Thanks for pointing that out.

Reply:
just released 3.0.1. And made a note to automate this, to have only one place to store the version string...

Reply:
👍